### PR TITLE
[hotfix][docs] Update the generated configuration docs for previous changes

### PR DESCRIPTION
**Pull request by zhijiangW**
## What is the purpose of the change

*Some configuration docs were forgot to update for previous changes, so run `generate-config-docs` for updating relevant ones.*

## Brief change log

  - *Update the generated blob_server_configuration.html*
  - *Add the new generated netty_shuffle_environment_configuration.html*

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by zhijiangW** [comment]
@flinkbot attention @azagrebin @zentol

**Comment by zhijiangW** [comment]
Thanks for the review @zentol !
I have addressed the above comments.

File: docs/ops/config.md [hunk 1]
```diff
@@ -100,9 +100,9 @@ The default fraction for managed memory can be adjusted using the taskmanager.me
 
 {% include generated/security_configuration.html %}
 
-### Network Environment
+### Netty Shuffle Environment
 
-{% include generated/network_environment_configuration.html %}
```
Comment (zentol) [inline docs/ops/config.md]: this file should also be deleted, I don't think this is used anywhere.

Comment (zhijiangW) [reply docs/ops/config.md]: fixed

**Pull request merged**